PF/Assignment1/assignment2.java

```diff
@@ -0,0 +1,52 @@
+package programing_fundamental;
+
+import java.util.*;
+
+public class assignment2 {
+	public int[] removeDuplicate(int input[])
+	{
+		int n=input.length-1;
+		
+		int i,j,k;
+		for(i=0;i<n;i++)
+		{
+			
+			for(j=i+1;j<=n;j++)
+			{
+				if(input[i]==input[j])
+				{
+					for(k=j;k<n;k++)
+						input[k]=input[k+1];
+					n--;
+					j--;
```

Comment:
ACTION: Instead of every time increasing and decreasing variable j in case of duplicate variable. Just increase it if input[i] is not equal to input[j] that means else case.